src/utils/invoice.ts

```
@@ -11,6 +11,11 @@ import { fetchBolt12Invoice } from "./boltzClient";
 import { lookup } from "./dnssec/dohLookup";
 import { checkResponse } from "./http";
 
+export const enum InvoiceType {
+    Bolt11,
+    Bolt12,
+}
```

Biome (1.9.4)

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)